src/core/server/bootstrap.ts

```diff
@@ -103,6 +103,9 @@ export async function bootstrap({
   try {
     await root.setup();
     await root.start();
+    if (cliArgs.dryRunMigration) {
+      return await shutdown();
+    }
```

Comment:
to allow more future cli tasks to exit after startup, maybe we need a `cliArgs.shutdownAfterStartup` that would currently only be set by `shutdownAfterStartup = dryRunMigration` ?